cmd/spire-server/cli/federation/refresh_test.go

```diff
@@ -34,17 +32,19 @@ func TestRefresh(t *testing.T) {
 		refreshResp *emptypb.Empty
 		serverErr   error
 
-		expectOut string
-		expectErr string
+		expectOutPretty string
+		expectOutJSON   string
+		expectErr       string
 	}{
 		{
 			name: "Success",
 			args: []string{"-id", "spiffe://example.org"},
 			expectReq: &trustdomainv1.RefreshBundleRequest{
 				TrustDomain: "spiffe://example.org",
 			},
-			expectOut:   "Bundle refreshed\n",
-			refreshResp: &emptypb.Empty{},
+			expectOutPretty: "Bundle refreshed\n",
+			expectOutJSON:   "{}",
```

Comment:
mmm not sure about this, 
empty json for a success sounds strange to me...
an options is to return a types.Status

Reply:
Yeah, so initially, I thought about mirroring the json output structure with the proto message definition, in a way that users already have documentation of what kind of data to expect by running each of the commands. Since this API call returns an `*emptypb.Empty` type, the final result was an empty json. Nevertheless, for this specific command, I don't see a problem of not mirroring the json response with the proto message, adding a status here would sure make things more clear, I will proceed with the update 😄

Reply:
d2a3aab